### ISSUE TITLE
When copy/paste same named file, new folder is created above working dir, dangerous

### ISSUE THREAD

**Issue by freed00m** [atom/tree-view#674]
Hi, I have a working directory, (project's directory openned with atom)

/home/frdm/gDrive/3.1__Tretak ZS/ZSUR/zsur-sp

And when I try copy+paste a file from sidebar tree-view, in my case file
```
/home/frdm/gDrive/3.1__Tretak ZS/ZSUR/zsur-sp/app/source/data.txt
```
to (note that it will not fail when no "data.txt" file is inside)
```
/home/frdm/gDrive/3.1__Tretak ZS/ZSUR/zsur-sp/tomas/Skripty/
```

What should happen?
Nothing or maybe warning or overwrite confirmation dialog. 

What happens instead.
A new directory tree is created outside working directory (note that data has no postfix ".txt")
```
/home/frdm/gDrive/30.1__Tretak ZS/ZSUR/zsur-sp/tomas/Skripty/data
```

From my point of view it is completely dangerous behavior.

---

Arch linux with a kernel 4.3.1 and I am using systemd 228

Atom version 1.3.1

**Comment by pydolan** [comment]
+1. This is really annoying.

**Comment by freed00m** [comment]
@pydolan yes, so when directory has number in them, it can happen above working dir. Lucky for us it does not overwrite another files, but can for some people create problems when random directories/files are spawned inside other projects.

**Comment by pydolan** [comment]
It looks like this may be related to #649.

**Comment by Ajedi32** [comment]
This problem manifests itself for me as an error:

![untitled](https://cloud.githubusercontent.com/assets/1876931/16696526/f0e5eb60-450a-11e6-9dd6-8487eba45db9.png)

And yes, it looks like #648 should fix it. See b6abd9cd3a58cae3fedd65c7eb5598fd5423f227.

Unfortunately it looks like Atom core is still using v0.205.0 of tree-view, so the bug isn't fixed there yet. It will be fixed once Atom upgrades to tree-view v0.207.0 or newer.

**Comment by 50Wliu** [comment]
Fixed by #649, should be in Atom 1.9.0.
